conf/config.neon

```diff
@@ -175,6 +175,7 @@ parameters:
 		- ../stubs/ext-ds.stub
 		- ../stubs/ImagickPixel.stub
 		- ../stubs/PDOStatement.stub
+		- ../stubs/PhpParserName.stub
```

Comment:
Please register this stub only in `build/phpstan.neon`. It's only for internal PHPStan purposes.